Quantum-Control-Applications/Superconducting/Single Fixed Transmon/T1.py

```diff
@@ -28,7 +29,7 @@
     with for_(n, 0, n < n_avg, n + 1):
         # Notice it's <= to include t_max (This is only for integers!)
         with for_(tau, tau_min, tau <= tau_max, tau + dtau):
-            play("pi", "qubit")
+            play("x180", "qubit")
```

Comment:
I must say that this "pi" vs "x180" is a bit confusing to me...
When do we want to use "pi" and when do we want to use "x180"?

Reply:
I would prefer `pi` here, but I didn't bother fighting over it...

Reply:
why would you prefer pi? If you use either x180 or y180 you get the same results. Actually in this program indeed you just need to start the state in the excited state, so maybe that tips the code towards 'pi' which is agnostic.

Reply:
Yea, I agree.
There are cases where it is clearly simpler to use `x180` and notation, for example, randomized benchmarking.
On the contrary, if you use frame rotation, it is clearer to use `pi`.
The question is what happens in the "simple" experiments such as Ramsey or T1.
I think that the fact that you don't care about the axis makes it better to use `pi`. This is what you won't say if you would describe the experiment in words.

Reply:
I like 'pi' in T1 experiments and when using frame rotations that is what I am going to do going forward.